src/main/java/com/jmb/springfactory/dao/productionorder/ProductionOrderMySQL.java

```diff
@@ -6,11 +6,14 @@
 import com.jmb.springfactory.exceptions.NotFoundException;
 import com.jmb.springfactory.exceptions.PersistenceLayerException;
 import com.jmb.springfactory.model.entity.ProductionOrder;
+import com.jmb.springfactory.model.enumeration.StatusEnum;
 
 public interface ProductionOrderMySQL extends GenericMySQLService<ProductionOrder, Integer> {
 
     Stream<ProductionOrder> findAllByScheduleId(Integer scheduleId);
 
     ProductionOrder save(ProductionOrder order, Integer scheduleId) throws PersistenceLayerException, NotFoundException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)